spec/features/request_options_spec.rb

```diff
@@ -22,10 +22,10 @@
       visit '/catalog/6890057'
     end
 
-    it 'does display a request button', unless: in_travis? do
+    it 'does not display a request button', unless: in_travis? || in_circleci? do
       sleep 5.seconds
       expect(page.all('.location--holding').length).to eq 1
-      expect(page.find_link('Request')).to be_visible
+      expect(page).not_to have_selector('.location-services.service-conditional')
```

Comment:
Out of curiosity, why was this affected by using CircleCI and Travis as opposed to just Travis?

Reply:
For this document https://catalog.princeton.edu/catalog/6890057 there is no 'Request' button.